src/refdb_fs.c

```diff
@@ -1269,37 +1286,48 @@ static int refdb_fs_backend__write(
 	const git_oid *old_id,
 	const char *old_target)
 {
-	refdb_fs_backend *backend = (refdb_fs_backend *)_backend;
 	git_filebuf file = GIT_FILEBUF_INIT;
+	git__refdb_flags flags = (force ? GIT_REFDB__FORCE_WRITE : 0);
 	int error = 0;
 
-	assert(backend);
-
-	if ((error = reference_path_available(backend, ref->name, NULL, force)) < 0)
-		return error;
-
-	/* We need to perform the reflog append and old value check under the ref's lock */
-	if ((error = loose_lock(&file, backend, ref->name)) < 0)
-		return error;
+	if ((error = refdb_fs_backend__write_tail(&file, _backend, ref, flags, old_id, old_target, who, message)) < 0) {
+		goto cleanup;
+	}
```

Comment:
Haha, you seem to have grown to like braces around single-line-statements :P

Reply:
Syntax is hard 😉. It's usually leftovers because of debugging code being present at some time.